build.sbt

```diff
@@ -98,6 +106,7 @@ val akkaClusterBootstrap = "com.lightbend.akka.management" %% "akka-management-c
 val akkaDiscoveryDns = "com.lightbend.akka.discovery" %% "akka-discovery-dns" % akkaManagementVersion
 val akkaDiscoveryMarathonApi = "com.lightbend.akka.discovery" %% "akka-discovery-marathon-api" % akkaManagementVersion
 val akkaManagementClusterHttp = "com.lightbend.akka.management" %% "akka-management-cluster-http" % akkaManagementVersion
+val reactiveLibServiceLocator = "com.lightbend.rp" %% "reactive-lib-service-discovery-lagom14-java" % "0.6.0"
```

Comment:
Released today with DC/OS support. Prefer this instead of `service-locator-dns` as it behaves the same as our Kubernetes stuff.